docs/source/diagrams/ReflectometryReductionOne_LowLvl-v1_wkflw.dot

```diff
@@ -75,23 +97,45 @@ $value_style
 
 inputWorkspace    -> checkXUnit
 checkXUnit        -> checkTransRun    [label="Yes"]
-checkXUnit        -> convertToWL      [label="No"]
-convertToWL       -> valMon           [label="Monitors"]
-convertToWL       -> valDet           [label="Detectors"]
-convertToWL       -> valUnity         [label="No Monitors"]
-valMon            -> checkNormDetMon
+checkXUnit        -> convertDet       [label="Detectors"]
+checkXUnit        -> checkMonUnity    [label="Monitors"]
+checkMonUnity     -> convertMon       [label="Yes"]
+checkMonUnity     -> valUnity         [label="No"]
+convertDet        -> valDet
+valDet            -> groupDet
+procCommands      -> groupDet
+convertMon        -> valMon
+monIntWavInterval -> intMon
```

Comment:
`intMon` is currently defined as:

`intMon [label="Integrate"]`

but the algorithm is `Integration`, so I think it should be defined as:

`intMon [label="Integration"]`